### ISSUE TITLE
Expect-CT hosts that aren't compliant: section 2.3.1

### ISSUE THREAD

**Issue by jcjones** [bifurcation/expect-ct#7]
Section **2.3.1.  Expect-CT Header Field Processing** says:

>If the header field contains the "enforce" directive and the connection does not comply with the UA's CT policy, then the UA MUST NOT note this host as an Expect-CT host.

My interpretation is that this intent is to not insta-brick the site if it's misconfigured. Perhaps we should clarify what behavior we do want, e.g.: treat it as if `enforce` was not present.

**Comment by jcjones** [comment]
Note also, **2.4.  Noting Expect-CT** is involved here:

> The UA MUST note a host as an Expect-CT host if and only if it received the Expect-CT response header field over an error-free TLS connection, including the validation added in Section 2.5, that included the "enforce" directive.

It's also true that the UA must not note under the circumstances in `2.3.1`. Perhaps the language from `2.3.1` should move here into `2.4` so that all "noting" behavior is together.

**Comment by estark37** [comment]
There's a paragraph below "If the header field contains..." that specifies what the UA should do if the 'enforce' directive isn't present (i.e., send a report). I think that paragraph probably wasn't there when you filed this issue.

I did some cleanup and consolidation of this section; in particular, I moved the requirement about non-compliant connections out of "Noting Expect-CT" and only mentioned it in passing, to make the "Header Field Processing" section the place that definitively states that requirement.
